homeassistant/components/fluss/button.py

```diff
@@ -0,0 +1,50 @@
+"""Support for Fluss Devices."""
+
+import logging
+
+from fluss_api.main import FlussApiClient
+
+from homeassistant.components.button import ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+LOGGER = logging.getLogger(__package__)
+DEFAULT_NAME = "Fluss +"
+
+
+class FlussButton(ButtonEntity):
+    """Representation of a Fluss cover device."""
+
+    def __init__(self, api: FlussApiClient, device: dict) -> None:
+        """Initialize the cover."""
+        self.api = api
+        self.device = device
+        self._name = device.get("deviceName", "Unknown Device")
+        self._attr_unique_id = f"fluss_{device.get('deviceId', 'unknown_id')}"
```

Comment:
When can a device id be not exisitng?

Reply:
Added a validation check to the button.py ensuring that the deviceId is required